packages/compose/src/hooks/use-copy-on-click/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import Clipboard from 'clipboard';
+
+/**
+ * WordPress dependencies
+ */
+import { useRef, useEffect, useState } from '@wordpress/element';
+
+/**
+ * Copies the text to the clipboard when the element is clicked.
+ *
+ * @param {Object}          ref     Reference with the element.
+ * @param {string|Function} text    The text to copy.
+ * @param {number}          timeout Optional timeout to reset the returned
+ *                                  state. 4 seconds by default.
```

Comment:
```suggestion
 *                                  state. 4000 milliseconds by default.
```

So that the expected unit is clear.